### PR TITLE
Fix IE exception when creating Url

### PR DESCRIPTION
**Pull request by XiaoxuHU**
Fix exception [[IE 11 Compatiality]Object doesn't support this action ](https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_workitems/edit/1495/)

image

### CODE REVIEW

**Comment by XiaoxuHU** [comment]
To fix this issue, I updated from core-js2 to core-js3. Since for core-js3 the file structure has changed, I need to update all import path within polyfill.ts file

File: AngularApp/projects/applens/src/polyfills.ts [hunk 1]
```diff
@@ -19,34 +19,34 @@
...
-import 'core-js/es7/array';
-import 'core-js/es7/string';
+import 'core-js/es/symbol';
+import 'core-js/es/object';
+import 'core-js/es/function';
+import 'core-js/es/parse-int';
+import 'core-js/es/parse-float';
+import 'core-js/es/number';
+import 'core-js/es/math';
+import 'core-js/es/string';
```
Comment (cindylovescoding) [inline AngularApp/projects/applens/src/polyfills.ts:29]: Remove duplicate import here.

**Comment by cindylovescoding** [comment]
Please do a basic checking and test, see if this is gonna introduce any regression

**Review by cindylovescoding — APPROVED** [review]

File: AngularApp/projects/applens/src/polyfills.ts (hunk 1 above)
Comment (XiaoxuHU) [reply AngularApp/projects/applens/src/polyfills.ts:29]: Done

**Comment by XiaoxuHU** [comment]
> Please do a basic checking and test, see if this is gonna introduce any regression

I tested, seems like everything works find in IE

**Pull request merged**